timev.rb

```diff
@@ -1,112 +1,124 @@
+#   # Seconds.
+#   Time.at(sec, in: tz) -> new_time
+#   # Milliseconds.
+#   Time.at(sec_i, msec, :millisecond, in: tz) -> new_time
+#   # Microseconds.
+#   Time.at(sec_i, usec,               in: tz) -> new_time
+#   Time.at(sec_i, usec, :usec,        in: tz) -> new_time
+#   Time.at(sec_i, usec, :microsecond, in: tz) -> new_time
+#   # Nanoseconds.
+#   Time.at(sec_i, nanoseconds, :nsec,       in: tz) -> new_time
+#   Time.at(sec_i, nanoseconds, :nanosecond, in: tz) -> new_time
+#
+# _Time_
+#
+# This form accepts a \Time object +time+
+# and optional keyword argument <tt>in: tz</tt>:
+#
+#   Time.at(Time.new)               # => 2021-04-26 08:52:31.6023486 -0500
+#   Time.at(Time.new, in: '+09:00') # => 2021-04-26 22:52:32.1480341 +0900
+#
+# _Seconds_
+#
+# This form accepts a numeric number of seconds +sec+
+# and optional keyword argument <tt>in: tz</tt>:
+#
+#   Time.at(946702800)               # => 1999-12-31 23:00:00 -0600
+#   Time.at(946702800, in: '+09:00') # => 2000-01-01 14:00:00 +0900
+#
+# _Milliseconds_
+#
+# This form accepts an integer number of seconds +sec_i+,
+# a numeric number of milliseconds +msec+,
+# a symbol argument +:millisecond+,
+# and an optional keyword argument <tt>in: tz</tt>:
+#
+#   Time.at(946702800, 500, :millisecond)               # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500, :millisecond, in: '+09:00') # => 2000-01-01 14:00:00.5 +0900
+#
+# _Microseconds_
+#
+# These forms accept an integer number of seconds +sec_i+,
+# a numeric number of microseconds +msec+,
+# an optional symbol +:usec+ or +:microsecond+,
+# and an optional keyword argument <tt>in: tz</tt>:
+#
+#   Time.at(946702800, 500000)                             # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000, :usec)                      # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000, :microsecond)               # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000, in: '+09:00')               # => 2000-01-01 14:00:00.5 +0900
+#   Time.at(946702800, 500000, :usec, in: '+09:00')        # => 2000-01-01 14:00:00.5 +0900
+#   Time.at(946702800, 500000, :microsecond, in: '+09:00') # => 2000-01-01 14:00:00.5 +0900
+#
+# _Nanoseconds_
+#
+# These forms accept an integer number of seconds +sec_i+,
+# a numeric number of nanoseconds +nsec+,
+# a symbol +:nsec+ or +:nanosecond+,
+# and an optional keyword argument <tt>in: tz</tt>:
+#
+#   Time.at(946702800, 500000000, :nsec)                     # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000000, :nanosecond)               # => 1999-12-31 23:00:00.5 -0600
+#   Time.at(946702800, 500000000, :nsec, in: '+09:00')       # => 2000-01-01 14:00:00.5 +0900
+#   Time.at(946702800, 500000000, :nanosecond, in: '+09:00') # => 2000-01-01 14:00:00.5 +0900
+#
+# Parameters:
+# :include: doc/time/sec_i.rdoc
+# :include: doc/time/msec.rdoc
+# :include: doc/time/usec.rdoc
+# :include: doc/time/nsec.rdoc
+# :include: doc/time/tz.rdoc
```

Comment:
Changing from `in: tz` to `in` will require changes here or to the included file, since the argument is `in`, not `tz`.

Reply:
Done.